model2/src/test/resources/META-INF/persistence.xml

```diff
@@ -35,7 +35,7 @@
         <properties>
             <property name="hibernate.connection.release_mode" value="after_transaction"/>
             <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect" />
-            <property name="hibernate.connection.url" value="jdbc:h2:./target/h2-syndesis"/>
+            <property name="hibernate.connection.url" value="jdbc:h2:mem:"/>
```

Comment:
switched to in-memory database to help tests run in parallel and not to trip over the same database file lock